invoke-function/handler.rb

```diff
@@ -0,0 +1,139 @@
+require 'net/http'
```

Comment:
Did you mean to commit or stash this?

Reply:
No this was not meant to be committed and has been removed.